### DISCUSSION TITLE
repo is code indexed but cannot be found by name

### DISCUSSION THREAD

**Discussion by TheDiscoMole** [community/community#56043] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My repository had it's code successfully indexed, but the the repository itself cannot be found through the navigation search.

**Comment by look** [comment]
Sorry you're having trouble with the search @TheDiscoMole. Can you clarify what you mean by "cannot be found through the navigation search"? How exactly are you trying to use search to navigate? If you can provide screenshots or videos, that would also be helpful.

**Comment by TheDiscoMole** [comment, marked as answer]
Thanks for the reply, after trying my query with other repositories `[repository-name] owner:[owner-name]` it appears that this query just doesn't work in general. Searching for my repository by description does seem to work fine, apologies for the inconvenience.

Though the above query not returning any relevant results seems a little counter intuitive.

**Reply by look** [reply]
It sounds like you're doing a search for repositories (not code), is that right?

In that case, try `[repository-name] org:owner-name` or `[repository-name] user:owner-name` (either works). `owner:` is not a configured qualifier for repository searches.

Here's an example search:

https://github.com/search?q=linguist+org%3Agithub-linguist&type=repositories
